### ISSUE TITLE
Server does not start. Error loading @serialport bindings file

### ISSUE THREAD

**Issue by sailoog** [SignalK/signalk-server-node#788]
New installation of 1.15.0 in Raspbian. This is the syslog, any idea?

May 12 17:23:44 openplotter signalk-server[1320]: Could not parse security config
May 12 17:23:44 openplotter signalk-server[1320]: ENOENT: no such file or directory, open '/home/pi/.signalk/security.json'
May 12 17:23:44 openplotter signalk-server[1320]: /usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/node_modules/bindings/bindings.js:135
May 12 17:23:44 openplotter signalk-server[1320]:   throw err;
May 12 17:23:44 openplotter signalk-server[1320]:   ^
May 12 17:23:44 openplotter signalk-server[1320]: Error: Could not locate the bindings file. Tried:
May 12 17:23:44 openplotter signalk-server[1320]:  → /usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/build/bindings.node
May 12 17:23:44 openplotter signalk-server[1320]:  → /usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/build/Debug/bindings.node
May 12 17:23:44 openplotter signalk-server[1320]:  → /usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/build/Release/bindings.node
May 12 17:23:44 openplotter signalk-server[1320]:  → /usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/out/Debug/bindings.node
May 12 17:23:44 openplotter signalk-server[1320]:  → /usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/Debug/bindings.node
May 12 17:23:44 openplotter signalk-server[1320]:  → /usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/out/Release/bindings.node
May 12 17:23:44 openplotter signalk-server[1320]:  → /usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/Release/bindings.node
May 12 17:23:44 openplotter signalk-server[1320]:  → /usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/build/default/bindings.node
May 12 17:23:44 openplotter signalk-server[1320]:  → /usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/compiled/8.16.0/linux/arm/bindings.node
May 12 17:23:44 openplotter signalk-server[1320]:  → /usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/addon-build/release/install-root/bindings.node
May 12 17:23:44 openplotter signalk-server[1320]:  → /usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/addon-build/debug/install-root/bindings.node
May 12 17:23:44 openplotter signalk-server[1320]:  → /usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/addon-build/default/install-root/bindings.node
May 12 17:23:44 openplotter signalk-server[1320]:  → /usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/lib/binding/node-v57-linux-arm/bindings.node
May 12 17:23:44 openplotter signalk-server[1320]:     at bindings (/usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/node_modules/bindings/bindings.js:126:9)
May 12 17:23:44 openplotter signalk-server[1320]:     at Object.<anonymous> (/usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/lib/linux.js:1:98)
May 12 17:23:44 openplotter signalk-server[1320]:     at Module._compile (module.js:653:30)
May 12 17:23:44 openplotter signalk-server[1320]:     at Object.Module._extensions..js (module.js:664:10)
May 12 17:23:44 openplotter signalk-server[1320]:     at Module.load (module.js:566:32)
May 12 17:23:44 openplotter signalk-server[1320]:     at tryModuleLoad (module.js:506:12)
May 12 17:23:44 openplotter signalk-server[1320]:     at Function.Module._load (module.js:498:3)
May 12 17:23:44 openplotter signalk-server[1320]:     at Module.require (module.js:597:17)
May 12 17:23:44 openplotter signalk-server[1320]:     at require (internal/module.js:11:18)
May 12 17:23:44 openplotter signalk-server[1320]:     at Object.<anonymous> (/usr/lib/node_modules/signalk-server/node_modules/@serialport/bindings/lib/index.js:14:22)
May 12 17:23:44 openplotter systemd[1]: signalk.service: Main process exited, code=exited, status=1/FAILURE
May 12 17:23:44 openplotter systemd[1]: signalk.service: Unit entered failed state.
May 12 17:23:44 openplotter systemd[1]: signalk.service: Failed with result 'exit-code'.
May 12 17:23:44 openplotter systemd[1]: signalk.service: Service hold-off time over, scheduling restart.
May 12 17:23:44 openplotter systemd[1]: Stopped signalk.service.
May 12 17:23:44 openplotter systemd[1]: signalk.service: Start request repeated too quickly.
May 12 17:23:44 openplotter systemd[1]: Failed to start signalk.service.
May 12 17:23:44 openplotter systemd[1]: signalk.socket: Unit entered failed state.
May 12 17:23:44 openplotter systemd[1]: signalk.service: Unit entered failed state.
May 12 17:23:44 openplotter systemd[1]: signalk.service: Failed with result 'exit-code'.

**Comment by tkurki** [comment]
Are you quite sure your installation is new?

I just did 
```
curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash - 
sudo apt-get install -y nodejs
sudo npm install -g --unsafe-perm signalk-server
signalk-server
```
and it works, including a NMEA0183 serial device.

Serialport is installed at `/usr/lib/node_modules/signalk-server/node_modules/serialport/`.

**Comment by sailoog** [comment]
SOLVED

I tried reinstalling serialport but I got errors so I deleted the serialport folders manually  from `/usr/lib/node_modules/signalk-server/node_modules/` and did `sudo npm install -g --unsafe-perm signalk-server` again. Then the error was in canboat and I did the same. Now is working ok.

I think it was a corrupted installation with some problem building some nodes. The super script to build OpenPlotter from scratch is about to be finished but it takes hours and this kind of error are expected.

https://github.com/openplotter/pi-gen/tree/openplotter

Thanks for pointing me to the problem.
